debian/control

```diff
@@ -8,6 +8,8 @@ Build-Depends-Indep: libglib2.0-dev,
                      meson (>= 0.60),
                      libxml2-utils,
                      sassc,
+                     inkscape,
+                     optipng,
```

Comment:
No please, if we need to generate something, let's do it in CI, to avoid having the builders to do all the job

Reply:
sure, will remove these